client/my-sites/github-deployments/components/deployments-survey/utils.ts

```diff
@@ -0,0 +1,29 @@
+import cookie from 'cookie';
+
+const setSurveyCookie = ( name: string, value: string, maxAge: number ) => {
+	document.cookie = cookie.serialize( `github-deployments-${ name }`, value, {
+		path: '/',
+		maxAge,
+	} );
+};
+
+const getSurveyCookie = ( name: string ) => {
+	return cookie.parse( document.cookie )[ `github-deployments-${ name }` ];
+};
```

Comment:
(Genuinely curious - not a criticism!) Is there any reason behind using cookies and not `savePreference` with a timestamp here? Wouldn't the latter apply to the user's account rather than just their browser?

Reply:
Personally I don't mind the survey showing again after someone time when the cookie expired, in this case after a year of the first survey popup. There isn't any strong opinion though, the survey modal component was already using cookie for its dismissed state instead of save preferences and I maintain that. But I wouldn't have a problem with the survey showing a couple times until we decide to close it.